### PR TITLE
Fix golint failures of e2e/framework/v*.go

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

This fixes golint failures of the following file:
- test/e2e/framework/volume_util.go

This changes following file also, because of changing function name in above file.
- test/e2e/storage/testsuites/volumes.go

Ref: #68026

**Special notes for your reviewer**:

None

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

**Comment by oomichi** [comment]
Thanks for doing this,

/ok-to-test

File: test/e2e/framework/volume_util.go [hunk 1]
```diff
@@ -53,29 +53,37 @@ import (
...
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
 )
 
 const (
+  // Kb is byte size of kilobyte
```
Comment (oomichi) [inline test/e2e/framework/volume_util.go]: The indent of this comment seems different from the actual code.
Is that just due to the web browser view? or actually different on the code?

Comment (atoato88) [reply test/e2e/framework/volume_util.go]: I'll fix this.

File: test/e2e/framework/volume_util.go [hunk 2]
```diff
@@ -174,7 +182,7 @@ func NewGlusterfsServer(cs clientset.Interface, namespace string) (config Volume
 	return config, pod, ip
 }
 
-// iSCSI-specific wrapper for CreateStorageServer.
+// NewISCSIServer is a iSCSI-specific wrapper for CreateStorageServer.
```
Comment (oomichi) [inline test/e2e/framework/volume_util.go]: nit: s/a/an/

Comment (atoato88) [reply test/e2e/framework/volume_util.go]: Right. I'll fix this.

**Review by neolit123 — COMMENTED** [review]
LGTM
thanks @atoato88

**Comment by oomichi** [comment]
Thanks for updating

/lgtm

**Review by neolit123 — COMMENTED** [review]
/approve

File: test/e2e/framework/volume_util.go [hunk 3]
```diff
@@ -473,24 +481,24 @@ func TestVolumeClient(client clientset.Interface, config VolumeTestConfig, fsGro
...
-			By("Checking fsType is correct.")
+			ginkgo.By("Checking fsType is correct.")
 			_, err = LookForStringInPodExec(config.Namespace, clientPod.Name, []string{"grep", " /opt/0 ", "/proc/mounts"}, fsType, time.Minute)
 			ExpectNoError(err, "failed: getting the right fsType %s", fsType)
 		}
 	}
 }
 
-// InjectHtml insert index.html with given content into given volume. It does so by
+// InjectHTML insert index.html with given content into given volume. It does so by
```
Comment (SataQiu) [inline test/e2e/framework/volume_util.go]: Um...  a little piece of advice, please change `insert` to `inserts`

Comment (atoato88) [reply test/e2e/framework/volume_util.go]: done.

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 

There are other parts of those files that make me cringe, but that's unrelated to this PR.

**Pull request merged**